### DISCUSSION TITLE
Cluster Not Coming up

### DISCUSSION THREAD

**Discussion by beegmon** [cloudnative-pg/cloudnative-pg#391] [Q&A]
I am trying to bring up a pretty simple test cluster of 3 replicas. The first 2 replicas come up fine, but the third just gets stuck. I don't see any errors in the operator logs but I do see these in the -join- job pod for the 3 replica

`{"level":"info","ts":1657246500.0914485,"msg":"Refreshed configuration file","logging_pod":"web3signerkeys-cluster-3","filename":"/controller/certificates/server.crt","secret":"web3signerkeys-cluster-server"}
{"level":"info","ts":1657246500.092129,"msg":"Refreshed configuration file","logging_pod":"web3signerkeys-cluster-3","filename":"/controller/certificates/server.key","secret":"web3signerkeys-cluster-server"}
{"level":"info","ts":1657246500.1031623,"msg":"Refreshed configuration file","logging_pod":"web3signerkeys-cluster-3","filename":"/controller/certificates/streaming_replica.crt","secret":"web3signerkeys-cluster-replication"}
{"level":"info","ts":1657246500.1037405,"msg":"Refreshed configuration file","logging_pod":"web3signerkeys-cluster-3","filename":"/controller/certificates/streaming_replica.key","secret":"web3signerkeys-cluster-replication"}
{"level":"info","ts":1657246500.1102498,"msg":"Refreshed configuration file","logging_pod":"web3signerkeys-cluster-3","filename":"/controller/certificates/client-ca.crt","secret":"web3signerkeys-cluster-ca"}
{"level":"info","ts":1657246500.118028,"msg":"Refreshed configuration file","logging_pod":"web3signerkeys-cluster-3","filename":"/controller/certificates/server-ca.crt","secret":"web3signerkeys-cluster-ca"}
{"level":"info","ts":1657246500.1181302,"msg":"Waiting for server to be available","logging_pod":"web3signerkeys-cluster-3","connectionString":"host=web3signerkeys-cluster-rw user=streaming_replica port=5432 sslkey=/controller/certificates/streaming_replica.key sslcert=/controller/certificates/streaming_replica.crt sslrootcert=/controller/certificates/server-ca.crt application_name=web3signerkeys-cluster-3 sslmode=verify-ca dbname=postgres connect_timeout=5 replication=1"}
{"level":"info","ts":1657246500.585573,"logger":"pg_basebackup","msg":"pg_basebackup: initiating base backup, waiting for checkpoint to complete","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246500.9173129,"logger":"pg_basebackup","msg":"pg_basebackup: checkpoint completed","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246500.9173608,"logger":"pg_basebackup","msg":"pg_basebackup: write-ahead log start point: 0/4000060 on timeline 1","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246500.9173667,"logger":"pg_basebackup","msg":"pg_basebackup: starting background WAL receiver","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246500.9583385,"logger":"pg_basebackup","msg":"pg_basebackup: created temporary replication slot \"pg_basebackup_300\"","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.4236193,"logger":"pg_basebackup","msg":"pg_basebackup: error: could not receive data from WAL stream: server closed the connection unexpectedly","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.4240875,"logger":"pg_basebackup","msg":"\tThis probably means the server terminated abnormally","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.4241407,"logger":"pg_basebackup","msg":"\tbefore or while processing the request.","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.4243414,"logger":"pg_basebackup","msg":"server closed the connection unexpectedly","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.4243531,"logger":"pg_basebackup","msg":"\tThis probably means the server terminated abnormally","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.4243577,"logger":"pg_basebackup","msg":"\tbefore or while processing the request.","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.424362,"logger":"pg_basebackup","msg":"server closed the connection unexpectedly","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.4244926,"logger":"pg_basebackup","msg":"\tThis probably means the server terminated abnormally","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
{"level":"info","ts":1657246804.4245908,"logger":"pg_basebackup","msg":"\tbefore or while processing the request.","pipe":"stderr","logging_pod":"web3signerkeys-cluster-3"}
`

**Comment by sxd** [comment]
@beegmon do you have the logs from the primary pod in this case? also a describe of the pod job will be great

**Reply by gbartolini** [reply]
Any news on sharing the logs?

**Reply by beegmon** [reply]
I haven't been able to recreate this faithfully yet. It happens sporadically.

I am going to be spinning up a few more clusters here in the next week or so and that will give me another chance to test/trigger this. i will be sure to grab the main pod and a describe
